### DISCUSSION TITLE
GitHub app logo changes not getting reflected

### DISCUSSION THREAD

**Discussion by jainankit** [community/community#24614] [Apps, API and Webhooks]
<p>Hey folks, I have a GitHub app where I updated the app-logo over 24 hours ago. It’s reflecting correctly when the app posts a comment on a PR but it still shows the old one on the apps public page. Can someone help?</p>

**Comment by jsoref** [comment]
<p>I’d file a ticket with</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://support.github.com">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/2119212/181085632-c9517f51-4a19-451f-9978-a34039c87a75.png" class="site-icon" width="32" height="32">

      <a href="https://support.github.com" target="_blank" rel="noopener">support.github.com</a>
  </header>

  <article class="onebox-body">
    

<h3><a href="https://support.github.com" target="_blank" rel="noopener">GitHub Support</a></h3>

  <p>GitHub Support is here to help. Learn about GitHub products, browse our helpful resources, and contact support with your questions.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by jainankit** [comment, marked as answer]
<p>Actually i figured out what was going on. So if the app is also registered in marketplace, the Logo for app home page is pulled from marketplace Logo (set separately), and the Logo that we see in app is configured separately. Wasn’t obvious.</p>
